apps/site/assets/ts/stop/components/DeparturesAndMap.tsx

```diff
@@ -62,16 +62,75 @@ const DeparturesAndMap = ({
     return false;
   };
 
+  const defaultPolylines = chain(routesWithPolylines)
+    .filter(
+      route =>
+        isASilverLineRoute(route) ||
+        isSubwayRoute(route) ||
+        isACommuterRailRoute(route)
+    )
+    .orderBy("sort_order", "desc")
+    .flatMap("polylines")
+    .uniqBy("id")
+    .value();
+
+  const DefaultRoutesMap = (): JSX.Element => {
+    return (
+      <StopMapRedesign stop={stop} lines={defaultPolylines} vehicles={[]} />
+    );
+  };
+
+  const SelectedRoutePatternMap = (): JSX.Element => {
```

Comment:
This could be broken out into its own file

Reply:
I'm torn -- it feels pretty tailored to the state in this component. Let's reconsider as we keep working in this component, or if we add more organization/hierarchy to the component files